examples/react-native/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "17.0.2",
+    "react": "^17.0.1",
```

Comment:
should remain, it needs to be exactly 17.0.2: https://www.runpkg.com/?react-native@0.68.0/package.json#86